docs/configuration_file.md

```diff
@@ -171,6 +173,12 @@ Some options are also mandatory.
 
 * `build_from` (*optional*, `string`) — build source to use, consists of 2 parts separated with delimiter ':', first part can be : image or imagestream, and second part is corresponding image or imagestream
 
+### `[platform:ARCH]` options
+
+* `architecture` (*optional*, `string`) — platform's GOARCH (Go language platform name). If not declared, this option assumes the value of `ARCH`.
```

Comment:
This reads a little bit as though the default is the literal string "ARCH". Can we re-word it to say the default is the same as the platform name being defined?

Reply:
done